tests/py/casmutils/xtal/coordinate.py.in

```diff
@@ -0,0 +1,91 @@
+#!@PYTHON@
+
+import unittest
+import casmutils as cu
+import numpy as np
+
+
+class CoordinateTest(unittest.TestCase):
+    def setUp(self):
+        """Setting up numpy arrays"""
+        self.lattice_matrix = np.array(
+            [[0, 0.5, 0.5], [0.5, 0, 0.5], [0.5, 0.5, 0]])
+        self.raw_coord0 = np.array([0.1, 0.2, 0.3])
+        self.raw_coord1 = np.array([0.2, 0.4, 0.6])
+
+        """Computing fractional coordinates manually"""
+        self.frac_coords0 = np.dot(np.linalg.inv(
+            self.lattice_matrix), self.raw_coord0)
+        self.frac_coords1 = np.dot(np.linalg.inv(
+            self.lattice_matrix), self.raw_coord1)
+
+        """Constructing Coordinate & MutableCoordinate classes"""
+        self.lat = cu.xtal.Lattice(*self.lattice_matrix)
```

Comment:
This block looks like it's testing the construction of the coordinates. Does this belong in `SetUp`, or should it be in `test_construct`? The purpose of the fixture is to systematically construct and tear down a collection of objects you can use in your tests. If you're not using variables in more than one test, it probably shouldn't be in `SetUp`.